### PR TITLE
fix for : Right clicking a log in content view should display a conte…

### LINKED ISSUE #57: 🐛Right clicking a log in content view should display a context menu with clickable items.
**:bug: Describe the bug**
If i right click on a log in the content view when no rows are selected, I expect the context menu to allow me to click on e.g. `Properties`. In order for this to work now, I first have to left click it in order to check the log, and then right click it.

**To Reproduce**
1. Navigate to a wellbore with log objects, so that the content view displays the logs with checkable rows.
2. Right click a log.
3. No actions are enabled in the context menu.

**Expected behavior**
I expect that all actions which can be done on a single log should be enabled, similar to when first left clicking the row.

**Screenshots**
How it looks now:

Screenshot 2020-12-18 at 11 34 46

How I suggest it should be:
Screenshot 2020-12-18 at 11 35 47

### PR DESCRIPTION
**Pull request by petrlicman**
[//]: # (Request Template Witsml Explorer)

[//]: # (Thank you for contributing to WITSML Explorer! Before submitting this PR, please fill in the following template.)

## Fixes

[//]: # (Write the GitHub issue number starting with #, or, for Equinor only, the Jira issue number starting with WE-)

This pull request fixes #57
Right clicking a log in content view should display a context menu with clickable items.

## Description

[//]: # (Please include a written summary of the changes and which issue is fixed. List any dependencies that are required for this change._)

## Type of change

[//]: # (Mark any of the types of change that apply.)

* [x] Bugfix
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Enhancement of existing functionality
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

## Impacted Areas in Application

[//]: # (List general components of the application that this PR will affect)

* [x] Frontend
* [ ] API
* [ ] WITSML
* [ ] Other (please describe)

## Checklist:

[//]: # (Please tick all the boxes or remove the ones that aren't needed and explain why)

*Communication*
* [ ] I have made corresponding changes to the documentation
* [ ] PR affects application security

*Code quality*
* [x] I have self-reviewed my code
* [x] No new warnings are generated

*Test coverage*
* [x] New code is covered by passing tests

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: Src/WitsmlExplorer.Frontend/components/ContentViews/table/ContentTable.tsx [hunk 1]
```diff
@@ -180,7 +180,10 @@ export const ContentTable = (contentTableProps: ContentTableProps): React.ReactE
                 <Fragment key={row.id}>
                   <StyledTr
                     selected={row.getIsSelected()}
-                    onContextMenu={(e) => onRowContextMenu(e, row)}
+                    onContextMenu={async (e) => {
+                      await row.toggleSelected(true);
```
Comment (janburak) [inline Src/WitsmlExplorer.Frontend/components/ContentViews/table/ContentTable.tsx:185]: I am getting a warning "'await' has no effect on the type of this expression.ts(80007)", but removing `await` make the context menu behave as if nothing is selected, so a comment would be nice to ensure that the `await` is not accidentally removed.
```suggestion
                      // await selection to ensure that the context menu detects that a row has been selected
                      await row.toggleSelected(true);
```

**Review by janburak — APPROVED** [review]
- [X] Read issue and PR
- [X] Pulled branch and manually tested
- [X] Verify that PR resolves issue
- [X] Reviewed the code

File: Src/WitsmlExplorer.Frontend/components/ContentViews/table/ContentTable.tsx (hunk 1 above)
Comment (petrlicman) [reply Src/WitsmlExplorer.Frontend/components/ContentViews/table/ContentTable.tsx:185]: Ok, I put the comment there

**Comment by janburak** [comment]
Please use topic branches on your fork for future PRs, rather than main, as specified in the [fork and pull request workflow](https://github.com/susam/gitpr#work-on-pull-request) which is our contributing process as specified in [CONTRIBUTING.MD](https://github.com/equinor/witsml-explorer/blob/main/CONTRIBUTING.md).

**Pull request merged**